policy/blacklist.go

```diff
@@ -1,1027 +0,0 @@
-// Copyright 2014 ISRG.  All rights reserved
```

Comment:
Please also move the PSL and PSL checking into this DB, since that will need updating as well.